beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/model/ExpiredEventModel.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.scheduler.apiary.model;
+
+import com.expediagroup.beekeeper.core.model.HousekeepingStatus;
+import com.expediagroup.beekeeper.core.model.LifecycleEventType;
+
+public class ExpiredEventModel implements EventModel {
+
+  private final LifecycleEventType lifecycleEventType;
+  private final HousekeepingStatus housekeepingStatus;
+
+  public ExpiredEventModel(LifecycleEventType lifecycleEvent, HousekeepingStatus housekeepingStatus) {
```

Comment:
Why does the `ExpiredEventModel` need the `housekeepingStatus`? Surely that's going to be `SCHEDULED`? Or am I missing something?

Reply:
Yup my bad. Removed it now